docs/troubleshoot/troubleshoot-zos-certificate.md

```diff
@@ -25,12 +25,57 @@ Dkeystore.pkcs12.keyProtectionAlgorithm=PBEWithSHAAnd3KeyTripleDES -J-
 Dkeystore.pkcs12.keyPbeIterationCount=50000
 ```
 
-- Set the flag `keystore.pkcs12.legacy` enabled with no value to create a PKCS12 keystore that can be loaded.
+- Set the flag `keystore.pkcs12.legacy` to enabled with no value to create a PKCS12 keystore that can be loaded.
 
-**Notes**
+:::note
+* If you already have an existing keystore or you are using keyrings, this error will not happen.
```

Comment:
I think the existing keystore part may need to clarify that it needs to be existing but created with a compatible java version

Reply:
Does this capture the addition: "Ensure that the existing keystore part is  created with a compatible java version."